index.js

```diff
@@ -0,0 +1,85 @@
+const project = [
+  {
+    image: './image/pic.jpeg',
+    title: 'Multi-Post Stories Gain+Glory',
+    language: ['HTML5', 'HTML5', 'HTML5'],
+    description: 'lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore distinctio aut illum officia unde, quod aliquam eos delectus aspernatur temporibus accusantium, sint quisquam sed quaerat provident tempora reprehenderit explicabo amet.',
+    liveLink: 'https://github.com/41hulk/micro-portfolio',
+    gitLink: 'https://github.com/41hulk/micro-portfolio',
+
+  },
+];
+
+const projectsDesktop = document.querySelector('.projects-desktop');
+
+for (let i = 0; i <= 5; i += 1) {
+  const card = document.createElement('div');
+  card.classList.add('project-container');
+
+  const img = document.createElement('img');
+  img.src = project[0].image;
+
+  const desc = document.createElement('div');
+  desc.classList.add('desc');
+
+  const title = document.createElement('h3');
+  title.innerHTML = project[0].title;
+
+  const languages = document.createElement('ul');
+
+  project[0].language.forEach((language) => {
+    const li = document.createElement('li');
+    li.innerHTML = language;
+    languages.appendChild(li);
+  });
+
+  const button = document.createElement('button');
+  const modal = document.querySelector('.modal-invisible');
+  button.setAttribute('type', 'button');
+  button.classList.add('project-button');
+  button.innerHTML = 'See project';
+
+  button.addEventListener('click', () => {
+    modal.style.display = 'block';
+    const content = `<div id="modal">
```

Comment:
- [ ] **Required:** There is yet some alignment of the button left to have a similar design as your chosen Figma Template.
- Kindly align the tags to left
- Also justify the buttons as shown in the images attached.

Your Design | Figma Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/151599840-56f62d4f-c9d7-463b-9914-75314ed5aa82.png) | ![image](https://user-images.githubusercontent.com/27709832/151600190-3cdc6916-8247-4674-a944-ecc97d2939cc.png)

- [ ] **Required:** Moreso, on the desktop popup
- The button text and image should match the template.
- The bottom should be aligned right as shown in the attached image.

Your Design | Figma Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/151600981-d69966f7-b56b-4501-9a92-c58d94ccb98c.png) | ![image](https://user-images.githubusercontent.com/27709832/151601159-f985f1a6-aea9-422a-a0c0-9aa9929ddca8.png)